plugins/jobs/src/js/jobsRunNow.js

```diff
@@ -0,0 +1,27 @@
+import "rxjs/add/operator/take";
+
+import { graphqlObservable } from "data-service";
+import gql from "graphql-tag";
+
+import defaultSchema from "./data/JobModel";
+
+const runNowJobMutation = gql`
+  mutation {
+    runJob(id: $jobId) {
+      jobId
+    }
+  }
+`;
+
+export default function jobsRunNow(jobId) {
+  return {
+    label: "Run Now",
+    onItemSelect() {
+      // take(1) makes sure the observable is going to complete after finishing
+      // the request, so we don't have to care about unsubscribing.
+      graphqlObservable(runNowJobMutation, defaultSchema, { jobId })
+        .take(1)
+        .subscribe();
```

Comment:
nothing? what about error handling? 🤔

Reply:
There was no errors being displayed in the previous implementation. If you click on a menu action that fails, you just do not see the results of it reflected on the UI.

Reply:
uh, ok :(